server/channels/app/support_packet.go

```diff
@@ -18,6 +21,10 @@ import (
 	"github.com/mattermost/mattermost/server/v8/config"
 )
 
+const (
+	cpuProfileDuration = 5 * time.Second
```

Comment:
I picked 5 seconds because it is a good tradeoff between wanting too long to get the support package and getting actual useful information out of the profile.

Reply:
@hanzei I think it would be better to use it in config.json. At times, there could be a need of data for longer duration.

Reply:
I'm hesitant to add yet another config value. The purpose of this feature is to collect information upfront and reduce the amount of round-trips between support and customer. There are surely times when we want more detailed information. But telling the customer which value they should use for the setting is similar to sending them the `curl` command that fetches the information directly.